metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1765,15 +1746,21 @@ def _exec_feature_query(self, *, feature_key: str) -> Dict:
 
         owners = self._create_owners(feature_records['owner_records'])
 
-        tags, owner_tags = self._classify_tags(feature_records.get('tag_records'))
+        tags = []
+        for record in feature_records.get('tag_records'):
+            tag_result = Tag(tag_name=record['key'],
+                             tag_type=record['tag_type'])
+            tags.append(tag_result)
 
         feature_node = feature_records['feat']
 
+        feature_group = feature_records['fg']
+        LOGGER.info(feature_node)
```

Comment:
does it make sense to log here? I'm not sure. If you are, let's at least add extra context.

Reply:
oops, forgot to clean up after debugging